apps/web/vibes/soul/form/input/index.tsx

```diff
@@ -27,19 +27,19 @@ import { Label } from '@/vibes/soul/form/label';
  */
 export const Input = React.forwardRef<
   React.ComponentRef<'input'>,
-  Omit<React.ComponentPropsWithoutRef<'input'>, 'id'> & {
+  React.ComponentPropsWithoutRef<'input'> & {
```

Comment:
Needed a way to use this component with a custom label, so I had to pass the `id` generated from conform-to. Curious why we omitted id in the first place.

Reply:
I think it's because we were passing hardcode IDs that would conflict. Your approach here is appropriate where we generate one if not provided one. The user of this component just needs to make sure to provide a unique ID as well, which I think conform does

Reply:
One more thing... this component should probably be renamed `InputField` as it's a form field, not just an input. And given that, we probably need to stop spreading props into the `<input>`.

Something to think about @hunterbecton, probably for after we release the Soul VIBE

Reply:
In this case, which is the correct Input to use?

Reply:
I think what you're doing here is right 👍 

No changes needed for this particular comment. My comments re: `InputField`, etc. were more for future changes to VIBES